content/source/docs/enterprise/vcs/troubleshooting.html.md

```diff
@@ -85,6 +85,20 @@ The domain name for Terraform Enterprise changed on 02/22 at 9AM from atlas.hash
 
 The fix is to update the OAuth Callback URL in your VCS provider to use app.terraform.io instead of atlas.hashicorp.com.
 
+### Changing the API URL for a VCS provider
```

Comment:
```suggestion
### Changing the URL for a VCS provider
```
I think people may need to change both the web and API URLs in some cases (in other cases, just API).